cli/cmd/lql.go

```diff
@@ -153,34 +159,41 @@ func init() {
 	// run specific flags
 	setQuerySourceFlags(queryRunCmd)
 
-	// since time flag
-	queryRunCmd.Flags().StringVarP(
+	// limit flag
+	queryRunCmd.Flags().IntVar(
+		&queryCmdState.Limit,
+		"limit", 0,
```

Comment:
Is this the only command we should add this new flag?

Reply:
Yes, my understanding of this flag (similar to the parameters) is that `limit` is only applicable to query execution

Reply:
I double checked the API docs and confirmed this.